bugbounty/settings.py

```diff
@@ -70,3 +73,21 @@
 
 # Slack for notifications
 SLACK_INCOMING_WEBHOOK = os.environ.get('SLACK_INCOMING_WEBHOOK', None)
+
+# python-social-auth config - use github
+AUTHENTICATION_BACKENDS = (
+    'social.backends.github.GithubTeamOAuth2',
+    'django.contrib.auth.backends.ModelBackend',
+)
+SOCIAL_AUTH_GITHUB_TEAM_KEY = os.environ.get('GITHUB_KEY', None)
+SOCIAL_AUTH_GITHUB_TEAM_SECRET = os.environ.get('GITHUB_SECRET', None)
+
+# The team ID doesn't seem to be easy to find on the web anywhere, so I
```

Comment:
agreed that it's hard to find. I added some docs in dolores on how I did it: https://github.com/18F/dolores-landingham-bot/compare/jy-github-auth#diff-6a3371457528722a734f3c51d9238c13R127

Not sure if that seems easier or harder than what you're recommending